DscResources/MSFT_xMySqlDatabase/MSFT_xMySqlDatabase.schema.mof

```diff
@@ -1,9 +1,10 @@
 [ClassVersion("1.0.0"),FriendlyName("xMySqlDatabase")] 
 class MSFT_xMySqlDatabase : OMI_BaseResource
 {
-    [Key, Description("Name of the database.")] String Name;
-    [Required, Description("Should the database be present or absent."), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
-    [Required, Description("The root credential that is used to install MySql server."), EmbeddedInstance("MSFT_Credential")] String ConnectionCredential;
+    [Key, Description("Name of the database.")] String DatabaseName;
```

Comment:
The resource didn't work before.  There are other changes that will break existing resources.  It is better to have clear names for things.  Also if you look at resources such as MSFT_GroupResource they also use a more descriptive name.